test/test_objects_skeleyaml.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+from skelebot.objects.skeleYaml import SkeleYaml
+
+class TChild(SkeleYaml):
+
+    par_one = None
+    par_two = None
+    nope = None
+
+    def __init__(self, par_one: int, par_two: int):
+        self.par_one = par_one
+        self.par_two = par_two
+        self.nope = "does not show"
+
+    def toDict(self):
+        self.nope = None
+        return super().toDict()
+
+class TParent(SkeleYaml):
+
+    childs = None
+    fav_child = None
+    
+    def __init__(self, childs: dict, fav_child: TChild):
+        self.childs = childs
+        self.fav_child = fav_child
+
+class TestSkeleYaml(unittest.TestCase):
+
+    def test_toDict(self):
+
+        parent = TParent({
+                "Jon": TChild(33, 1),
+                "Geoffrey": TChild(45, None)
+            }, TChild(12, 42))
+
+        res = parent.toDict()
+        res_childs = res["childs"]
+        res_jon = res_childs["Jon"]
+        res_geo = res_childs["Geoffrey"]
+        res_fav = res["fav_child"]
+
+        self.assertEqual(res_jon["par_one"], 33)
+        self.assertEqual(res_jon["par_two"], 1)
+        self.assertEqual(res_jon.get("nope", None), None)
```

Comment:
```suggestion
        self.assertNotIn("nope", res_jon)
```
I think this check makes more sense. It could be that the key actually exists with a value of None.
Same with the other None checks below

Reply:
Ah, that's just cleaner too. Thanks, I'll update.